app/containers/App.jsx

```diff
@@ -4,11 +4,18 @@ import { renderRoutes } from 'react-router-config';
 import { Header } from '../components/Common/Header';
 import { Footer } from '../components/Common/Footer';
 import { TrackingScript } from '../components/Common/TrackingScript';
+import { isClient } from '../config/app';
 import '../sass/global/styles.scss';
 import './App.scss';
 
-// eslint-disable-next-line
 class App extends Component {
+  componentDidUpdate(prevProps) {
+    const { location } = this.props;
+    const { location: prevLocation } = prevProps;
+    if (isClient && (prevLocation.pathname !== location.pathName)) {
```

Comment:
Have you tested this locally? I have no idea how this could work given your comparing `pathname` to `pathName` (capital N).

My guess is that currently each time App renders the page will jump to the top as undefined doesn't equal whatever URL is the correct spelling of the variable.